pumpkindb_engine/src/lib.rs

```diff
@@ -3,10 +3,9 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
-#![feature(slice_patterns, advanced_slice_patterns)]
+#![feature(slice_patterns, raw_vec_internals)]
 
 #![cfg_attr(test, feature(test))]
-#![cfg_attr(not(target_os = "windows"), feature(alloc))]
 #![feature(alloc)]
```

Comment:
This is stabilized on nightly.

Reply:
Hi @bjorn3, I tried to make a minimal patch to get this project to build. There are quite a number of warning about stabilised features and deprecated modules. Do you think I should try to resolve them in this PR?

Reply:
I don't know, I am not a maintainer of this project.